tensorflow/contrib/data/python/ops/dataset_ops.py

```diff
@@ -1879,7 +1879,8 @@ def output_types(self):
 
 def _should_unpack_args(args):
   """Returns `True` if `args` should be `*args` when passed to a callable."""
-  return nest.is_sequence(args) and not isinstance(args, dict)
+  is_namedtuple = isinstance(args, tuple) and hasattr(args, "_fields")
+  return nest.is_sequence(args) and not isinstance(args, dict) and not is_namedtuple
```

Comment:
Perhaps it would be simpler to replace this with `return type(args) is tuple`? (We might need a pylint exclusion to make the sanity check presubmit happy, but I think it would be justified in this case.)